.changeset/hip-tables-check.md

```diff
@@ -0,0 +1,13 @@
+---
+"@latticexyz/store": major
+"@latticexyz/world": major
```

Comment:
```suggestion
"@latticexyz/world": patch
```
As far as I can tell, this didn't break any world APIs, this is just a bump of the store dep?

Reply:
yeah, keyswithvalue got an internal breaking change, but that's not api